code/game/machinery/autolathe.dm

```diff
@@ -118,13 +118,19 @@
 		return 1
 
 	if (src.m_amount + O.m_amt > max_m_amount)
-		user << "<span class=\"alert\">The autolathe is full. Please remove metal from the autolathe in order to insert more.</span>"
+		user << "<span class='warning'>The autolathe is full. Please remove metal from the autolathe in order to insert more.</span>"
 		return 1
 	if (src.g_amount + O.g_amt > max_g_amount)
-		user << "<span class=\"alert\">The autolathe is full. Please remove glass from the autolathe in order to insert more.</span>"
+		user << "<span class='warning'>The autolathe is full. Please remove glass from the autolathe in order to insert more.</span>"
 		return 1
-	if (O.m_amt == 0 && O.g_amt == 0)
-		user << "<span class=\"alert\">This object does not contain significant amounts of metal or glass, or cannot be accepted by the autolathe due to size or hazardous materials.</span>"
+	if (!O.m_amt && !O.g_amt)
+		user << "<span class='warning'>This object does not contain sufficient amounts of metal or glass to be accepted by the autolathe.</span>"
+		return 1
+	if(istype(user, /mob/living/silicon))

```

Comment:
Borg modules all get NODROP anyway, so couldn't you just take this out and let the other check handle it? Would be a bit cleaner.